src/components/__stories__/login_header.tsx

```diff
@@ -5,5 +5,5 @@ import LoginHeader from "../login_header"
 
 storiesOf("LoginHeader", LoginHeader)
   .add("Simple login header", () => (
-    <LoginHeader/>
+    <LoginHeader title={"Welcome back, please log in to your account"}/>
```

Comment:
If you look at the DOM you’ll see that there’s now 1 more wrapping div than you probably intended:

<img width="592" alt="screen shot 2017-03-01 at 19 54 20" src="https://cloud.githubusercontent.com/assets/2320/23475804/f6c3df52-feb8-11e6-810f-a235fc470729.png">